tools/opt/opt.cpp

```diff
@@ -590,28 +592,9 @@ int main(int argc, const char** argv) {
     return 1;
   }
 
-  if (!skip_validator) {
-    // Let's do validation first.
-    spv_context context = spvContextCreate(target_env);
-    spv_diagnostic diagnostic = nullptr;
-    spv_const_binary_t binary_struct = {binary.data(), binary.size()};
-    spv_result_t error =
-        spvValidateWithOptions(context, options, &binary_struct, &diagnostic);
-    if (error) {
-      spvDiagnosticPrint(diagnostic);
```

Comment:
@dj2 This how the messages were printed before.  If there was an error, the diagnostic we passed to `spvDiagnosticPrint`, which always prints to stderr.  This is fine for code specific to spirv-opt, but I do not want to call if from public API.

Reply:
Right, makes sense. Thanks for pointing that out.